packages/components/dropdown/src/utils.ts

```diff
@@ -35,6 +35,18 @@ const getElementId = (element?: ReactElement) => {
   return element ? (element.type as FC & { id?: string }).id : ''
 }
 
+export const findElement =
```

Comment:
I think we should have some kind of generic "utils" package in order to consume it from any component, instead of creating this functions locally, WDYT?

Reply:
yes, I agree. Its repeated in many places

Reply:
- chip
- select
- dropdown
- input (not exactly the same function but this can work too for its purpose)